src/pages/events/eventsInfo/submission/viewForm/ViewStepper.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import {Box, Step, StepConnector, Stepper} from '@mui/material'
+import useViewForm from '../../../../../assets/styleJs/viewForm/viewForm'
+
+const ViewStepper = () => {
+  const classes = useViewForm()
+  const steps = [
+    {
+      id: 1,
+    },
+    {
+      id: 2,
+    },
+    {
+      id: 3,
+    },
+    {
+      id: 4,
+    },
+  ]
+  return (
+    <Box>
+      <Stepper
+        activeStep={4}
```

Comment:
Why active step is 4, can we store steps somewhere else?

Reply:
It's ok